packages/circularprogress/src/react/index.tsx

```diff
@@ -62,6 +62,7 @@ const styles: { [key: string]: StyleFn } = {
 interface CircularProgressProps extends HTMLPropsFor<'div'> {
   size?: ValueOf<typeof vars.sizes>
   value?: number
+  'aria-label': string
```

Comment:
Now required = breaking change. But I don't see the commit message indicating this.

Also, I see a "--wip--" commit that needs reworded to conventional commit or dropped.

Reply:
My commits are always squashed I'll add breaking to the last one. I honestly don't think require something that should be there for a11y is breaking.

Reply:
If someone runs tsc in their build, in practical terms it will be breaking.

Reply:
Also, squashed is fine to get rid of the WIP commit. But still want to be able to gen new versions for both circularprogress and linearprogress packages. That will require 2 commits.